zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -519,6 +519,13 @@ For more information about SLF4J, see
 For more information about Logback, see
 [Logback website](http://logback.qos.ch/).
 
+#### Logging in JSON format
+
+Through its pluggable and flexible log framework, Zookeeper can easily be configured to log in JSON format.
+An example of JSON logging using the popular [ECS schema](https://doc.wikimedia.org/ecs/) is provided as a logback appender in `logback.xml`.
+You can test it by enabling the `CONSOLE-JSON` appender, e.g. by starting Zookeeper with
+the system property: `-Dzookeeper.log.appender=CONSOLE-JSON`.
```

Comment:
JSON formatting is a niche use case. This is one of millions of possible ways of configuring logging. I'm not sure it warrants a special call out in the ZK docs. The previous paragraph already links to the logback website, if people want to do something custom with logback.